src/main/scala/com/codacy/parsers/implementation/CoberturaParser.scala

```diff
@@ -79,16 +79,20 @@ class CoberturaParser(val language: Language.Value, val rootProject: File, val c
         key -> value
     }
 
-    allFiles.find(f => f.getAbsolutePath.endsWith(sourceFilename)).map {
+    allFiles.map(f => sanitiseFilename(f.getAbsolutePath)).find(f => f.endsWith(sourceFilename)).map {
       filename =>
-        CoverageFileReport(sanitiseFilename(filename.getAbsolutePath), fileHit, lineHitMap)
+        CoverageFileReport(stripRoot(filename), fileHit, lineHitMap)
     }
   }
 
   private def sanitiseFilename(filename: String): String = {
-    filename.stripPrefix(rootProjectDir)
+    filename
       .replaceAll("""\\/""", "/") // Fix for paths with \/
-      .replace("\\", "/") // Fix for paths with \\
+      .replace("\\", "/") // Fix for paths with \
```

Comment:
Note that I corrected the inaccurate comment here. The windows parsing actually relies on the behaviour as it exists rather than as it is commented, so I wanted to be clear that this was (at least for my use case) the desired functionality.